.github/workflows/nightly.yaml

```diff
@@ -35,7 +35,9 @@ jobs:
         registry-url: 'https://registry.npmjs.org'
 
     - name: Install dependencies
-      run: yarn install --immutable --inline-builds --mode=skip-build
+      run: |
+        yarn install --immutable --inline-builds --mode=skip-build
+        git checkout yarn-project.nix
```

Comment:
Nice hack :grin:
@YorikSar, it looks like the nix plugin doesn't respect the `--immutable` flag FYI

Reply:
I think this means that `yarn-project.nix` got diverged from the state of `yarn.lock`, and that should raise red flags, because during Nix build only `yarn-project.nix` is used.

Did you check what diff it generates? Maybe we need to commit it instead?

Reply:
This happens in `nightly` branch which has it's own `yarn.lock` as it diverges from `master` by bumping versions of internal cross-package dependencies. `--immutable` makes it not update `yarn.lock`, but nix plugin does not respect this cli arg.